tests/adapters/switches/juniper_test.py

```diff
@@ -2553,7 +2732,7 @@ def test_unset_interface_access_vlan_on_trunk_mode_raises(self):
 
         assert_that(str(expect.exception), contains_string("Operation cannot be performed on a trunk mode interface"))
 
-    def test_unset_interface_access_vlan_on_unknown_interface_raises(self):
+    def test_unset_interface_access_vlan_on_default_interface_works(self):
```

Comment:
Didn't you remove a valid test here, shouldn't the default interface be a separate test?